tests/modules/dryrun.c

```diff
@@ -0,0 +1,43 @@
+
+#include "redismodule.h"
+#include <errno.h>
+#include <assert.h>
+#include <string.h>
+#include <strings.h>
+
+/* A wrap for SET command with ACL check on the key. */
+int dryrun(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
```

Comment:
i'm not sure we need a new unit and a new module for this.
i think it can be achieved with the `test_rm_call_flags` command in misc.c
i.e. the tcl test code can pass the 'D` flag, and there's no need for a new module or a new command.

Reply:
yes, I figured that out after writing this test.  will redo with that.